src/AsyncWriterResult/Library.fs

```diff
@@ -262,6 +288,8 @@ type AsyncWriterResultBuilder() =
     member __.ReturnFrom(m: Async<Writer<'w, Result<'a, 'b>>>) = m
     member __.Bind(m, f) = AsyncWriterResult.bind f m
     member __.Zero() = __.Return()
+    member __.BindReturn(x, f) = AsyncWriterResult.map f x

```

Comment:
didn't realise `BindReturn` is just `map`!

is it possible to use `map` "on its own" in a CE? as in, does the `!` syntax support it?

Reply:
Sorry not quite sure what you mean by this?